extensions/Worker.Extensions.ServiceBus/src/ServiceBusMessageActionsConverter.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.ServiceBus.Grpc;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Converter to bind to <see cref="ServiceBusMessageActions" /> type parameter.
+    /// </summary>
+    internal class ServiceBusMessageActionsConverter : IInputConverter
```

Comment:
I don't think this uses deferred binding and so does not need the `[SupportsDeferredBinding]` attribute, but just want to double check that's correct?

Reply:
Correct, this is just for binding to the client-like type ServiceBusMessageActions.